Library/Formula/unittest-cpp.rb

```diff
@@ -2,20 +2,11 @@
 
 class UnittestCpp < Formula
   homepage 'http://unittest-cpp.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/unittest-cpp/UnitTest++/1.4/unittest-cpp-1.4.zip'
```

Comment:
We don't accept ` head `-only formula in the core. We'll either need a stable release, or we'll need to migrate this to ` Homebrew-head-only `.